client/cmd/dexc/ui/config.go

```diff
@@ -56,19 +58,23 @@ func setNet(net string) string {
 
 // Config is the configuration for the DEX client application.
 type Config struct {
-	DataDir    string `long:"dir" description:"Path to application directory"`
-	Config     string `long:"config" description:"Path to an INI configuration file. default:[home]/dexc_mainnet.conf"`
-	DBPath     string `long:"db" description:"Database filepath. Database will be created if it does not exist."`
-	CertsPath  string `long:"certs" description:"Path to a JSON-formatted file linking DEX URL keys to TLS certificate filepaths."`
-	RPCOn      bool   `long:"rpc" description:"turn on the rpc server"`
-	RPCAddr    string `long:"rpcaddr" description:"RPCServer listen address"`
-	WebOn      bool   `long:"web" description:"turn on the web server"`
-	WebAddr    string `long:"webaddr" description:"HTTP server address"`
-	NoTUI      bool   `long:"notui" description:"disable the terminal-based user interface. must be used with --rpc or --web"`
-	Testnet    bool   `long:"testnet" description:"use testnet"`
-	Simnet     bool   `long:"simnet" description:"use simnet"`
-	ReloadHTML bool   `long:"reload-html" description:"Reload the webserver's page template with every request. For development purposes."`
-	DebugLevel string `long:"log" description:"Logging level {trace, debug, info, warn, error, critical}"`
+	DataDir     string `long:"dir" description:"Path to application directory"`
```

Comment:
Another consistency point, and I know this wasn't a change in your PR, but `appdata` has a known meaning (different between platforms, but always a subdirectory in the user's home directory), so I would go with that.

Also, the default path being a `dexclient` subfolder is pretty confusing: https://github.com/decred/dcrdex/blob/13a6d7b7fb69c59115d99199f57147eee93bc28c/client/cmd/dexc/ui/config.go#L33

On nix, I would expect `dexc` to have a default appdata path of `~/.dexc`.  @buck54321, were there other considerations here, or would that change be agreeable?

Reply:
The `dir` config name was rubbing me wrong too. Not sure why I did that. `appdata` is a better flag name, and `dexc` default directory name is better too.

Reply:
I changed both the config field and the flag name to AppData, appdata.